lib/cartopy/tests/mpl/test_features.py

```diff
@@ -49,6 +47,7 @@ def test_natural_earth_custom():
     return ax.figure
 
 
+@pytest.mark.skipif(not _HAS_PYKDTREE_OR_SCIPY, reason='pykdtree or scipy is required')
```

Comment:
This skip would be unnecessary if the test didn't use `stock_img`, which I don't think the test really needs.  However I'm not sure it's worth replacing the reference image for the purpose of this exercise.